package.json

```diff
@@ -4,6 +4,6 @@
     "cucumber": "^7.0.0-rc.0",
     "dotenv": "^16.0.1",
     "isomorphic-fetch": "^3.0.0",
-    "pg": "^8.8.0"
+    "pg": "^8.11.0"
   }
```

Comment:
I'm not sure what the influence of this could be on other components.

Reply:
@bscheltinga Yes, I need to remove this. Somehow these changes always happen for me without me doing anything.